vevestaX/vevesta.py

```diff
@@ -61,7 +61,7 @@ def get_filename(self):
         return filename
 
     def __find_access_token(self):
-        token = None
+        token = ""
```

Comment:
Why have you replace TOken None to "" ?

Reply:
to avoid any null error while sending it to java backend. plus, we set git_token to empty string in __find_git_token ( https://github.com/Vevesta/VevestaX/blob/d31b4e2c4f0c778cfb9487d801233acc064bdba8/vevestaX/vevesta.py#L107 ). so i wanted to follow similar pattern for access token.